app/proprietary/src/main/java/stirling/software/proprietary/security/model/AuthenticationType.java

```diff
@@ -2,5 +2,8 @@
 
 public enum AuthenticationType {
     WEB,
-    SSO
+    @Deprecated(since = "1.0.2")
```

Comment:
Wasn't sure if this should still be kept or not, after the changes in this PR `OAUTH2` or `SAML2` are used in its place and it doesn't seem to be used anywhere else